### PR TITLE
Create Blog “call-for-hpe-ezmeral-unified-analytics-beta-testers”

### PR DESCRIPTION
**Pull request by drensing**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+**Let your opinion be known!**
+
+One could consider HPE Ezmeral Unified Analytics as an opinionated set of software. Part of the purpose of this beta is to ensure that our opinion makes sense. We are trying to select the most important open source products in data engineering, data analytics, and data science. Because we don’t expect our customers to choose all of our tools, we designed our workflows and pipelines to allow easy replacement of an included component by an alternative. We try to provide a comprehensive selection of products, but we also try to stay out of your way if you have preferences. For example, while we include Super Set for building reports and dashboards, if you prefer to use Tableau, our JDBC Presto API’s are available for a quick swap. If you prefer using Weights and Bias’, you can swap out our ML Flow registry solution. Models can be published to Kserve or Sagemaker Endpoints.
```
Comment (choukden) [inline content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md:21]: For example, while we include Super Set for building....

Previous occurrence of Apache Superset was in one word (SuperSet).
Maybe for consistency purpose, spell it the same throughout the post.

Same remark for "you can swap out our ML Flow registry solution". Is it "ML Flow" or "MLFlow" registry ? Please confirm.

File: content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**Sign up today**
+
+The HPE Ezmeral portfolio of software products provides businesses with the foundation needed to develop analytic applications across hybrid-cloud environments. HPE Ezmeral lets you seamlessly access, manage and govern all your data, regardless of where it is located, and combine it into a single view. Combined with our open-source analytics framework that allows your developers to use the tools of their choice, high-performance analytics can be delivered at enterprise scale with an overall lower TCO.
+
+The HPE Ezmeral Early Access beta program allows developers to try these new products before they’re released and thereby influence the product’s development. The HPE Ezmeral Unified Analytics beta operationalizes hybrid multi-cloud modern analytic and advanced analytic workloads and lets you choose where you deploy your compute and data. Register today to learn more about how you can participate in this program.
```
Comment (choukden) [inline content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md:29]: Register today to learn more....

I assume that a registration link will be added here. Correct?

**Review by choukden — CHANGES_REQUESTED** [review]
Just a couple of minor comments.

Thanks.
Denis.

File: content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+As part of the beta program, you will have access to an HPE Ezmeral Unified Analytics environment hosted on HPE infrastructure. We will provide you with initial training to get you up to speed on the various open source frameworks that are included. We will also provide you a Slack channel for support questions. We will rely on you to provide feedback so we can change direction, if necessary. This initial beta test will be available for about 3 months and will then move to a stage where you will be able to deploy HPE Ezmeral Unified Analytics in your own environment for additional testing and feedback. During and immediately after the initial beta period, we will request feedback informally and through surveys.
```
Comment (choukden) [inline content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md:23]: Slack Channel....do we know the link for the slack channel? Once we know it, we could add it as hotlink.

File: content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md (hunk 1 above)
Comment (drensing) [reply content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md:21]: Apache Superset and MLflow... I figured out how to change it

File: content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md (hunk 3 above)
Comment (drensing) [reply content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md:23]: I don't have the link for it yet, but yes, I put the words in here so we could link it once we knew it. Unfortunately, I'm not sure how to add links in the editing piece.

File: content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md (hunk 2 above)
Comment (drensing) [reply content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md:29]: Yes, that is correct. I did add in that sentence you suggested.

**Pull request merged**